apitools/base/py/credentials_lib_test.py

```diff
@@ -63,5 +63,38 @@ def testGceServiceAccounts(self):
     self._GetServiceCreds(service_account_name='my_service_account')
 
 
+class Test__GetRunFlowFlags(unittest2.TestCase):
```

Comment:
you can just call it `TestGetRunFlowFlags` -- easier to read and unlikely to be confusing.